src/Compilers/VisualBasic/Portable/PublicAPI.Unshipped.txt

```diff
@@ -32,32 +31,43 @@ Microsoft.CodeAnalysis.VisualBasic.Syntax.TupleTypeSyntax.Update(openParenToken
-Shared Microsoft.CodeAnalysis.VisualBasic.SyntaxFactory.TupleElement(type As Microsoft.CodeAnalysis.VisualBasic.Syntax.TypeSyntax) -> Microsoft.CodeAnalysis.VisualBasic.Syntax.TupleElementSyntax
+Overrides Microsoft.CodeAnalysis.VisualBasic.VisualBasicSyntaxRewriter.VisitTypedTupleElement(node As Microsoft.CodeAnalysis.VisualBasic.Syntax.TypedTupleElementSyntax) -> Microsoft.CodeAnalysis.SyntaxNode
+Shared Microsoft.CodeAnalysis.VisualBasic.SyntaxFactory.NamedTupleElement(identifier As Microsoft.CodeAnalysis.VisualBasic.Syntax.IdentifierNameSyntax) -> Microsoft.CodeAnalysis.VisualBasic.Syntax.NamedTupleElementSyntax
+Shared Microsoft.CodeAnalysis.VisualBasic.SyntaxFactory.NamedTupleElement(identifier As Microsoft.CodeAnalysis.VisualBasic.Syntax.IdentifierNameSyntax, asClause As Microsoft.CodeAnalysis.VisualBasic.Syntax.SimpleAsClauseSyntax) -> Microsoft.CodeAnalysis.VisualBasic.Syntax.NamedTupleElementSyntax
 Shared Microsoft.CodeAnalysis.VisualBasic.SyntaxFactory.TupleExpression(ParamArray arguments As Microsoft.CodeAnalysis.VisualBasic.Syntax.SimpleArgumentSyntax()) -> Microsoft.CodeAnalysis.VisualBasic.Syntax.TupleExpressionSyntax
 Shared Microsoft.CodeAnalysis.VisualBasic.SyntaxFactory.TupleExpression(arguments As Microsoft.CodeAnalysis.SeparatedSyntaxList(Of Microsoft.CodeAnalysis.VisualBasic.Syntax.SimpleArgumentSyntax)) -> Microsoft.CodeAnalysis.VisualBasic.Syntax.TupleExpressionSyntax
 Shared Microsoft.CodeAnalysis.VisualBasic.SyntaxFactory.TupleExpression(openParenToken As Microsoft.CodeAnalysis.SyntaxToken, arguments As Microsoft.CodeAnalysis.SeparatedSyntaxList(Of Microsoft.CodeAnalysis.VisualBasic.Syntax.SimpleArgumentSyntax), closeParenToken As Microsoft.CodeAnalysis.SyntaxToken) -> Microsoft.CodeAnalysis.VisualBasic.Syntax.TupleExpressionSyntax
 Shared Microsoft.CodeAnalysis.VisualBasic.SyntaxFactory.TupleType(ParamArray elements As Microsoft.CodeAnalysis.VisualBasic.Syntax.TupleElementSyntax()) -> Microsoft.CodeAnalysis.VisualBasic.Syntax.TupleTypeSyntax
 Shared Microsoft.CodeAnalysis.VisualBasic.SyntaxFactory.TupleType(elements As Microsoft.CodeAnalysis.SeparatedSyntaxList(Of Microsoft.CodeAnalysis.VisualBasic.Syntax.TupleElementSyntax)) -> Microsoft.CodeAnalysis.VisualBasic.Syntax.TupleTypeSyntax
 Shared Microsoft.CodeAnalysis.VisualBasic.SyntaxFactory.TupleType(openParenToken As Microsoft.CodeAnalysis.SyntaxToken, elements As Microsoft.CodeAnalysis.SeparatedSyntaxList(Of Microsoft.CodeAnalysis.VisualBasic.Syntax.TupleElementSyntax), closeParenToken As Microsoft.CodeAnalysis.SyntaxToken) -> Microsoft.CodeAnalysis.VisualBasic.Syntax.TupleTypeSyntax
+Shared Microsoft.CodeAnalysis.VisualBasic.SyntaxFactory.TypedTupleElement() -> Microsoft.CodeAnalysis.VisualBasic.Syntax.TypedTupleElementSyntax
```

Comment:
>Shared Microsoft.CodeAnalysis.VisualBasic.SyntaxFactory.TypedTupleElement() [](start = 0, length = 75)

I believe we shouldn't have this method in the API, it can never represent a valid syntax.